README.md

```diff
@@ -270,9 +270,10 @@ Use this repository: https://github.com/yb85/scantailor-advanced-osx
 Build instructions for Qt 6
 ----------
 
-Unfortunately, this project does not support the build of scantailor-advanced with Qt6 yet. It is however possible to build it from the command line, starting with a precompiled version of Qt.
+It is possible to build scantailor-advanced with Qt6. However, scantailor-libs-build does not include tools for this yet.
+The following instructions assume a precompiled version of Qt.
 
-Please note, that Qt 6 doesn't support Windows 7.
+Please note that Qt 6 doesn't support Windows <= 7.
```

Comment:
```suggestion
Please note that Qt 6 doesn't support Windows 7 or older.
```
I believe that would be easier to understand.

Reply:
Agreed.